pallets/runtime/common/src/dividend.rs

```diff
@@ -1,463 +0,0 @@
-// This file is part of the Polymesh distribution (https://github.com/PolymathNetwork/Polymesh).
```

Comment:
`create_at_by` in `checkpoints.rs` should be made private again now to resolve the TODO.

Reply:
Ok.